style.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+}
+
+body {
+  overflow-x: hidden;
+  max-width: 100vh;
+}
+
+header {
+  background: #fff;
+  width: 100%;
+  height: 70px;
+  display: flex;
+  align-items: center;
+  padding: 0 30px;
+}
+
+.my_logo a {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  user-select: none;
+  cursor: pointer;
+  font-size: 25px;
+}
+
+nav {
+  width: 50px;
+  height: auto;
+  position: absolute;
+  right: 0;
+  margin: 0 30px;
+  cursor: pointer;
+}
+
+nav .bar {
+  width: 30px;
+  height: 4px;
+  margin: 4px 0;
+  border-radius: 2px;
+  background: #6070ff;
+}
+
+.container {
+  background-image: url('Images/background_img.png');
+  background-repeat: no-repeat;
+  max-width: 100%;
+  height: auto;
+  margin-top: 30px;
+  left: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+.header-text {
+  width: auto;
+  height: auto;
+  margin: 0 24px;
+}
+
+.header-text h2 {
+  color: #000;
```

Comment:
- [Optional] kindly, use the Figma text color, for your heading which is `color: #172B4D;` o fit the Figma design👍👌.